build.gradle.kts

```diff
@@ -236,7 +236,6 @@ subprojects {
       exceptionFormat = TestExceptionFormat.FULL
       showStandardStreams = false
     }
-    dependsOn(":startRedis")
```

Comment:
Starting Redis via Docker before running any test is annoying locally. I think I've added it as a dependency in the modules that require it which has sped up local development already.